be/src/exec/schema_scanner/schema_partitions_meta_scanner.cpp

```diff
@@ -261,8 +262,8 @@ Status SchemaPartitionsMetaScanner::fill_chunk(ChunkPtr* chunk) {
         }
         case 22: {
             // DATA_SIZE
-            Slice data_size = Slice(info.data_size);
-            fill_column_with_slot<TYPE_VARCHAR>(column.get(), (void*)&data_size);
+            int64_t data_size_bytes = parse_data_size(info.data_size);
```

Comment:
better to remove `ByteSizeValue` conversion in FE, so than we can remove such complex conversion in later version.

Reply:
> better to remove `ByteSizeValue` conversion in FE, so than we can remove such complex conversion in later version.

update.